### ISSUE TITLE
Add `false` to list of ignored values

### ISSUE THREAD

**Issue by alizahid** [dcastil/tailwind-merge#24]
Great package! I'm switching over from `clsx` because of the obvious.

Can we add `false` to the list of [ignored values](https://github.com/dcastil/tailwind-merge#ignores-undefined-and-null-values)?

It would allow me to do this without ESLint warnings;

```typescript
<div className={twMerge('flex', className, foo && 'border-2')} />
```

**Comment by dcastil** [comment]
Hey @alizahid! I thought about this, too. Currently I use `args.join(' ')` internally which only filters `null` and `undefined`, but not `false`.

I'll change that and release a v0.7.0 soon. 😊

**Comment by dcastil** [comment]
Just released this in [`v0.7.0`](https://github.com/dcastil/tailwind-merge/releases/tag/v0.7.0).

**Comment by dcastil** [comment]
Just released this in [`v0.7.0`](https://github.com/dcastil/tailwind-merge/releases/tag/v0.7.0).

**Comment by alizahid** [comment]
Amazing, thanks!
